wp-parsely.php

```diff
@@ -476,6 +495,25 @@ public function insert_parsely_page() {
         return $parselyPage;
     }
 
+	/**
+    * Add Google AMP tracking code
+    */
+    public function insert_parsely_amp_tracking() {
+        $parselyOptions = $this->get_options();
+        // If we don't have an API key, there's no need to proceed.
+        if ( empty($parselyOptions['apikey']) ) {
+		  return;
+        }
+        global $post;
+        $display = TRUE;
+        if ( is_single() && $post->post_status != 'publish' ) {
+            $display = FALSE;
+        }
+        if ( $display ) {
+            include('parsely-amp-tracking.php');
+        }
+    }
+
```

Comment:
This whole function can be deleted. Let's instead make this the place in `wp-parsely.php` where we define our new `public function parsely_add_amp_actions()` and `public function parsely_add_amp_analytics($analytics)`